Projects/Scripts/Misc/Email.cs

```diff
@@ -126,7 +126,7 @@ private static async void SendAsync(MimeMessage message)
         catch (Exception exception)
         {
           Console.WriteLine(exception);
-          if (m_retryCount >= 0)
+          if (RETRY_SEND_EMAIL_COUNT >= 0)

```

Comment:
I am still not sure how useful this is.
I would rather we console out only ONCE (w/ stacktrace).
Then use a finally block to console out saying we failed all retries and it was never sent.

Should we talk about intermittent service failures in general that are non-fatal and how to handle them?
I assume there are libraries to handle this, but I actually don't have much experience with the new C# ecosystem.

Reply:
I gonna provide another console handler with some cosmetics like yellow warning output and red stacktraces and so on. Like I say, I think its ok for this pull request because it also relates to the whole project.